build.sbt

```diff
@@ -27,24 +27,24 @@ lazy val schemaExtender = Projects.schemaExtender
 
 dependsOn(domainClasses)
 libraryDependencies ++= Seq(
-  "com.github.pathikrit" %% "better-files"  % "3.9.2",
-  "com.github.scopt"     %% "scopt"         % "4.1.0",
-  "io.joern"             %% "x2cpg"         % Versions.joern,
-  "io.joern"             %% "x2cpg"         % Versions.joern % Test classifier "tests",
-  "io.joern"             %% "javasrc2cpg"   % Versions.joern,
```

Comment:
is this formatting change expected?

Reply:
`scalafmt` would've caught it, haven't done any change manually.